src/main/kotlin/com/sourcegraph/cody/statusbar/CodyStatusBarWidget.kt

```diff
@@ -20,7 +21,9 @@ class CodyStatusBarWidget(project: Project) : EditorBasedStatusBarPopup(project,
     if (currentStatus == CodyStatus.CodyDisabled) {
       return WidgetState.HIDDEN
     }
-    val state = WidgetState(currentStatus.presentableText, "", true)
+    // Remote environment does not support rendering icons, so we use a placeholder text instead
+    val text = if (GraphicsUtil.isRemoteEnvironment()) "Cody" else ""
```

Comment:
@dominiccooney Looks like the remote checks I suggested to you before does not work as I expected, but this does.